desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -130,10 +147,64 @@ void TabDiveInformation::divesChanged(dive_trip *trip, const QVector<dive *> &di
 	case DiveField::WATER_TEMP:
 		ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 		break;
+	case DiveField::ATM_PRESS:
+		ui->atmPressVal->setText(ui->atmPressVal->text().sprintf("%d",current_dive->surface_pressure.mbar));
+		break;
 	case DiveField::DATETIME:
 		updateWhen();
 		break;
 	default:
 		break;
 	}
 }
+
+void TabDiveInformation::updateTextBox(int i)
+{
+	pressure_t atmpress = { 0 };
+	double altitudeVal;
+	if (current_dive) {
+		switch (ui->atmPressType->currentIndex())
+		case 0:		// If an atm pressure has been specified in mbar:
+			if (i == 1) {	// this is only triggered by on_atmPressVal_editingFinished()
```

Comment:
Something is very weird here -> this braces should be after the switch!

The fact that this compiles is scary and amazing, but the way C (and by inheritance C++) works!

Reply:
Corrected